docs/data/joy/components/accordion/accordion.md

```diff
@@ -0,0 +1,50 @@
+---
+product: joy-ui
+title: React Accordion component
+githubLabel: 'component: accordion'
+waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/accordion/
+---
+
+# Accordion
+
+<p class="description">The accordion component allows the user to show and hide sections of related content on a page.</p>
+
+:::success
+Joy Accordion component is still **in development** but it should not block you from using Joy UI!
+
+The examples below demonstrate the featuring between Joy UI and popular headless libraries.
+:::
+
+If you need Joy UI Accordion, please upvote [this issue](https://github.com/mui/material-ui/issues/36281). It will help us prioritize the next component that we should work on.
+
+## Integration
+
+Joy [`List`](/joy-ui/react-list/) components are perfect for building accordions because they are vertically stacked with the same size.
+
+- **List**: Wraps the accordions and set `component="div"` to remove it from the accessibility tree.
```

Comment:
Let's use "component case" for these, as with the [MUI Base Tabs](https://mui.com/base/react-tabs/).

```suggestion
- `<List />` - Wraps the accordion and sets `component="div"` to remove it from the accessibility tree.
```

Reply:
Noted